src/store/reducers.ts

```diff
@@ -291,15 +291,32 @@ export const errorsReducer = handleActions(
 
 
 // FIXME Move all the below (potentially the fetchCount ones too) to their own file
+export interface TaskSettings {
+  showCompleted: boolean;
+  sortOrder: string;
+  filterBy: string;
+  searchTerm: string;
```

Comment:
This should not be a setting, it's ephemeral, not persistent... It should just be a prop / navigation property.

Reply:
The reason I have these as settings is that I want the filtering/sorting to persist when the user navigates away from the Tasks page. The main use case is when a user goes to edit/create a task. After saving and navigating back to the tasks page, all the sorting/filtering is reset, which can be frustrating.

Do you have a suggestion for another way I can handle this?

Reply:
Sorry if my initial comment wasn't clear, I was referring specifically to searchTerm, not the rest. The rest should indeed be as you did them. As I told you elsewhere, it's similar to how I did it in etesync-tasks, which you can refer to if you want a glimpse into my thoughts process on things, and anyhow, a lot of it is very similar. Obviously my thoughts aren't necessarily good, but at least you can see what I mean. :)